projects/core/src/error-handling/effects-error-handler/index.ts

```diff
@@ -0,0 +1,9 @@
+/*
+ * SPDX-FileCopyrightText: 2024 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+export * from '../../model/error-action';
```

Comment:
[MINOR]
It looks strange it's exported from folder `../../model` - this folder contains mostly backend-related models. Moreover the ErrorAction is only related to ngrx. and only related to our error handling model.

For easier maintenance, I can see now it would be better to place it here in this folder.

PS. It's only an internal structural change that doesn't impact the public API, so it's not urgent before FF.